app/models/person.rb

```diff
@@ -23,10 +23,18 @@ def full_name
     "#{title} #{name}"
   end
 
-  def to_string
+  def to_s
     full_name
   end
 
+  def related_searchable_records
+    if room
+      chairs + [room]
```

Comment:
![image](https://user-images.githubusercontent.com/68114173/152161094-985f12a0-3c50-44b7-bcda-ef92524b256e.png)
Why is the room condition necessary here bot not chairs (or likewise people and rooms for the chair `related_searchable_records` method)

Reply:
I think the problem was that a person has at most one room. In the other cases we get a list of chairs/people/rooms. So we had to make a list out of the room and this did not work, wenn there was no room.